### PR TITLE
GEODE:9009 finish support for functionality compatible with Redis DEC…

### PR DESCRIPTION
**Pull request by jhutchison**
…RBY command

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java [hunk 1]
```diff
@@ -50,48 +52,99 @@ public void tearDown() {
...
+  public void shouldThrowError_givenWrongNumberOfArguments() {
+    assertExactNumberOfArgs(jedis, Protocol.Command.DECRBY, 2);
   }
 
   @Test
   public void decrbyMoreThanMaxLongThrowsArithmeticException() {
     jedis.set("somekey", "1");
+
+    BigInteger maxLongValue = new BigInteger(String.valueOf(Long.MAX_VALUE));
+    BigInteger biggerThanMinLongValue = maxLongValue.add(new BigInteger("1"));
```
Comment (jdeppe-pivotal) [inline geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java:108]: This should probably be called `biggerThanMaxLongValue`

**Review by jdeppe-pivotal — APPROVED** [review]
LGTM, other than minor variable naming request.

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java (hunk 1 above)
Comment (jhutchison) [reply geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java]: yes it should.  thanks.  changed.

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java [hunk 2]
```diff
@@ -33,14 +34,15 @@
 public abstract class AbstractDecrByIntegrationTest implements RedisPortSupplier {
 
   private Jedis jedis;
-  private Random rand;
+  private Jedis jedis2;
```
Comment (sabbey37) [inline geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java:37]: If we end up adding the additional test, could we also remove `jedis2`? it is not used.

**Review by sabbey37 — CHANGES_REQUESTED** [review]
One additional scenario that would be good to add to the integration tests is when a key is already set to the `Long.MIN_VALUE` and we try to `DECRBY` 1.  It should result in the error message: `ERR increment or decrement would overflow`.  You could also test setting a key to `Long.MAX_VALUE` and doing `DECRBY` -1 to make sure we get the `ERR increment or decrement would overflow` message (maybe the second scenario is slightly redundant though).

File: geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java (hunk 2 above)
Comment (jhutchison) [reply geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java]: whoops-  thanks

Comment (jhutchison) [reply geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractDecrByIntegrationTest.java]: added test

**Review by sabbey37 — APPROVED** [review]
Looks good!

**Comment by sabbey37** [comment]
DUnit test failure is unrelated to this PR.

**Pull request merged**